test/blackbox-tests/test-cases/coq/coqtop-root.t/run.t

```diff
@@ -14,5 +14,5 @@ This test is currently broken due to the workspace resolution being faulty #5899
   1 | (coq.theory
   2 |  (name foo))
   Error: 'coq.theory' is available only when coq is enabled in the dune-project
-  file. You must enable it using (using coq 0.6) in your dune-project file.
+  file. You must enable it using (using coq 0.7) in your dune-project file.
```

Comment:
```suggestion
  file. You must enable it using (using Coq 0.7) in your ``dune-project`` file.
```

Reply:
This is the `(using coq 0.7)` stanza from dune-project. Since it is the output of the command line, there is no point putting monospace either.